jena-kafka-connector/src/main/java/org/apache/jena/kafka/KafkaConnectorAssembler.java

```diff
@@ -83,57 +81,61 @@ public class KafkaConnectorAssembler extends AssemblerBase implements Assembler
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConnectorAssembler.class);
 
     private static String NS = "http://jena.apache.org/fuseki/kafka#";
-    public static String getNS() { return NS; }
 
-    /** Type of a connector description */
-    public static Resource tKafkaConnector = ResourceFactory.createResource(NS+"Connector");
+    public static String getNS() {
+        return NS;
+    }
+
+    /**
+     * Type of a connector description
```

Comment:
```suggestion
     * Type of connector description
```
Utterly pedantic - fixing grammatical error